macros/LearnSidebar.ejs

```diff
@@ -152,6 +139,23 @@ var text = mdn.localStringMap({
         'Client-side_web_APIs_Drawing_graphics': 'Drawing graphics',
         'Client-side_web_APIs_Video_and_audio_APIs': 'Video and audio APIs',
         'Client-side_web_APIs_client-side_storage': 'Client-side storage',
+    'Web_forms' : 'Web forms — Collecting data from users',
```

Comment:
I wouldn't name this "Collecting data from users" given how data collection is quite negatively connoted these days. How about "Web forms – Working with user data" ? Just a thought, though.